Something/Service/Services/Company/CompanyService.cs

```diff
@@ -17,17 +17,17 @@ public CompanyService(ApplicationDbContext dbContext)
             _dbContext = dbContext;
         }
 
-        public IEnumerable<CompanyViewModel> GetCompanies(Guid userId)
+        public IEnumerable<CompanyVM> GetCompanies(Guid userId)
         {
             var companies = _dbContext
                                     .CompanyUsers
                                     .Where(x => x.UserId == userId)
                                     .Select(x => x.Company).ToList();
 
-            var companiesVm = new List<CompanyViewModel>();
+            var companiesVm = new List<CompanyVM>();
             foreach (var company in companies)
             {
-                companiesVm.Add(new CompanyViewModel(company.CompanyId, company.Name));
+                companiesVm.Add(new CompanyVM(company.CompanyId, company.Name));
```

Comment:
For now this is fine, but as stuff gets more complex it would make sense to have a mapper service

Reply:
FYI creating a companiesVM can be done via a LINQ through.
`var companiesVM = companies.Select(c => new CompanyVM(*****)).ToList())` no need of foreach.